crates/burn-dataset/Cargo.toml

```diff
@@ -33,6 +33,8 @@ dataframe = ["dep:polars"]
 burn-common = { path = "../burn-common", version = "0.17.0", optional = true, features = [
     "network",
 ] }
+burn-tensor = { path = "../burn-tensor", version = "0.17.0" }
+burn-ndarray = { path = "../burn-ndarray", version = "0.17.0" }
```

Comment:
I would move these under `[dev-dependencies]` since they'll be needed for tests.

Reply:
Will do